### PR TITLE
Issue 108: Adding bubblescan exceutable bash script

### LINKED ISSUE #108: Package BubbleScan as a Desktop Application
Description:
Create a desktop version of the BubbleScan web app using tools like PyInstaller or Electron to bundle the Flask server and frontend into a self-contained desktop application.

Tasks:

- Set up the Flask application to run as a local server for the desktop version.
- Package the application using PyInstaller or Electron.
- Ensure all necessary dependencies are included in the package.

Success Criteria:

- [x] The application runs as a standalone desktop app without requiring users to install Flask separately.
- [x] The packaged app launches the local server and frontend successfully on different operating systems 
- [x] Verify the app's functionalities (file upload, custom sheets) work correctly in the desktop version.

### PR DESCRIPTION
**Pull request by mylescorrea**
Fixes #108 

**What was changed?**

*Created a bash script that creates the bubblescan executable in the dist folder inside of the ServerCode Folder*

**Why was it changed?**

*This was changed because we need to run our entire program with just one .exe file*

**How was it changed?**

*I changed the appserver.py file to handle opening browsers. Then using pyinstaller (added to the requirements.txt), I can create a bash script that creates this .exe and runs our program with one click. *

**How to run the script**

```bash

chmod +x Bubblescan.sh
./Bubblescan.sh

```
**Bubblescan.exe will be in the dist folder in BubbleScan/ServerCode/dist**

### CODE REVIEW

File: ServerCode/application/AppServer.py [hunk 1]
```diff
@@ -286,4 +290,12 @@ def csv_acknowledgment(self, file_id):
 app_server = AppServer(app)
 
 if __name__ == '__main__':
-    app_server.app.run(host='0.0.0.0', port=5001, debug=True)
+    # Delay to ensure the server is up
+    time.sleep(2)
```
Comment (kate-holdener) [inline ServerCode/application/AppServer.py]: Relying on time.sleep to synchronize processes is always a bad idea.

Comment (mylescorrea) [reply ServerCode/application/AppServer.py]: No problem will delete it

**Comment by Chirag2x** [comment]
@mylescorrea, This is the good work. Can you also update the readme file to add the above information with all the steps. That would be great for anyone new to access and use the application.

File: BubbleScan.sh
Comment (Chirag2x) [inline BubbleScan.sh:1]: This bash file handle all the components. Good work here.

File: README.md
Comment (Chirag2x) [inline README.md:1]: This is really important to update the Readme file for new folks.

**Review by Chirag2x — APPROVED** [review]

**Pull request merged**